pkg/datapath/iptables/iptables.go

```diff
@@ -1235,7 +1233,8 @@ func (m *Manager) installMasqueradeRules(
 				if cidr.Equal(r.Dst, cidr.ZeroNet(r.Family)) {
 					progArgs = append(
 						progArgs,
-						"!", "-d", snatDstExclusionCIDR)
+						"!", "-d", strings.Join(snatDstExclusionCIDRs, ","),
```

Comment:
I have been using iptables my entire professional life and I didn't realize you could comma-separate multiple cidrs :exploding_head: